src/utils-matrix.jl

```diff
@@ -321,13 +317,13 @@ function b3_f(
            ) +
            L_m * rho_x * V / (rc * V * temp) *
            (deps_x_dmh2o_t * dM_h2o_t_dt + deps_x_dmco2_t * dM_co2_t_dt) +
```

Comment:
7/14 12:20pm Chris said this sign should be `-`
```suggestion
           (deps_x_dmh2o_t * dM_h2o_t_dt + deps_x_dmco2_t * dM_co2_t_dt) -
```